presto-main/src/test/java/com/facebook/presto/sql/planner/iterative/rule/TestMergeDuplicateAggregation.java

```diff
@@ -0,0 +1,51 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.sql.planner.iterative.rule;
+
+import com.facebook.presto.sql.planner.assertions.PlanMatchPattern;
+import com.facebook.presto.sql.planner.iterative.rule.test.BaseRuleTest;
+import com.google.common.collect.ImmutableList;
+import com.google.common.collect.ImmutableMap;
+import org.testng.annotations.Test;
+
+import static com.facebook.presto.SystemSessionProperties.MERGE_DUPLICATE_AGGREGATIONS;
+import static com.facebook.presto.common.type.BigintType.BIGINT;
+import static com.facebook.presto.sql.planner.assertions.PlanMatchPattern.aggregation;
+import static com.facebook.presto.sql.planner.assertions.PlanMatchPattern.expression;
+import static com.facebook.presto.sql.planner.assertions.PlanMatchPattern.project;
+import static com.facebook.presto.sql.planner.assertions.PlanMatchPattern.values;
+
+public class TestMergeDuplicateAggregation
+        extends BaseRuleTest
+{
+    @Test
+    public void testMergeAggregations()
+    {
+        tester().assertThat(new MergeDuplicateAggregation(getMetadata().getFunctionAndTypeManager()))
+                .setSystemProperty(MERGE_DUPLICATE_AGGREGATIONS, "true")
+                .on(p -> p.aggregation(af -> {
+                    p.variable("col", BIGINT);
+                    af.globalGrouping()
+                            .addAggregation(p.variable("sum_1"), p.rowExpression("sum(col)"))
```

Comment:
can you also add sum() on a different column to show that it doesn't get merged, and maybe a different agg on the same column to show the same?